src/app/security-group/sg-filter/sg-filter.component.ts

```diff
@@ -44,4 +44,18 @@ export class SgFilterComponent {
   public get SecurityGroupViewMode() {
     return SecurityGroupViewMode;
   }
+
+  public ngOnChanges(changes: SimpleChanges) {
+    const accounts = changes['accounts'];
+    if (accounts) {
+      this.onAccountQueryChanged(this.accountQuery);
```

Comment:
`ERROR in src/app/security-group/sg-filter/sg-filter.component.ts(51,39): error TS2339: Property 'accountQuery' does not exist on type 'SgFilterComponent'.`